### DISCUSSION TITLE
[@types/react-transition-group] @types/react@18 breaks @types/react-transition-group

### DISCUSSION THREAD

**Discussion by herrlegno** [DefinitelyTyped/DefinitelyTyped#59874] [Issues with a @types package] [answered]
@types/react-transition-group uses:
```json
"dependencies": {
    "@types/react": "*"
},
```

[Reference](https://github.com/facebook/react/issues/24304)

[a comment by an automated account is not shown]

**Comment by herrlegno** [comment, marked as answer]
Bumping with the errors I get after a Yarn Install

EDIT: way too long error list. Solution in the reply of this comment

**Reply by ybiquitous** [reply]
The workaround on https://github.com/facebook/react/issues/24304#issuecomment-1094565891 may help you.

See also https://github.com/facebook/react/issues/24304#issuecomment-1111472024 for more details.
